### PR TITLE
Make lesson links relative ... and correct.

### PR DESCRIPTION
**Pull request by adamhooper**
I spotted styling inconsistencies and incorrect links in English. I
guessed that the same occurred in Greek, and that the fixes are the
same.

Now, all inter-lesson links are relative (e.g.,
`<a href="scrape-using-xpath">`. This wasn't an option for us
previously, when lessons URLs used Django's oddball URL scheme (in which
each URL ends in `/`). But it's an option now, and I'm glad to declare
relative URLs the de-facto means of linking from one lesson to another.

### CODE REVIEW

File: server/lessons/en/custom-alerts.html [hunk 1]
```diff
@@ -73,10 +73,10 @@ <h2>Lesson complete!</h2>
     <h4>Continue learning:</h4>
     <ol>
       <li>
-        <a href="/lessons/scrape-table/">Scrape and monitor a table from a web page.</a>
+        <a href="scrape-table">Scrape and monitor a table from a web page.</a>
       </li>
       <li>
-        <a href="/lessons/clean-and-standardize/">Scrape a webpage to extract or monitor content</a>
+        <a href="clean-and-standardize">Scrape a webpage to extract or monitor content</a>
```
Comment (giorgio93p) [inline server/lessons/en/custom-alerts.html]: The text should be "Clean and standardize data"

File: server/lessons/el/prosarmosmenes-eidopoihseis.html [hunk 2]
```diff
@@ -60,8 +60,8 @@ <h2>Εδώ τελειώνει το μάθημα!</h2>
   <div class="next">
     <h4>Συνεχίστε τα μαθήματα:</h4>
     <ol>
-      <li><a href="/lessons/el/syllogh-pinaka/">Συλλογή και παρακολούθηση πίνακα από ιστοσελίδα</a></li>
-      <li><a href="/lessons/el/kathariste-vromika-dedomena/">Συλλέξτε περιεχόμενο από μια ιστοσελίδα και παρακολουθήστε το</a></li>
+      <li><a href="syllogh-pinaka">Συλλογή και παρακολούθηση πίνακα από ιστοσελίδα</a></li>
+      <li><a href="kathariste-vromika-dedomena">Συλλέξτε περιεχόμενο από μια ιστοσελίδα και παρακολουθήστε το</a></li>
```
Comment (giorgio93p) [inline server/lessons/el/prosarmosmenes-eidopoihseis.html]: The text in `<a href="kathariste-vromika-dedomena"></a>` should be "Καθαρίστε και τυποποιήστε δεδομένα"

File: server/lessons/el/analysh-dedomenwn-twitter.html [hunk 3]
```diff
@@ -118,8 +118,8 @@ <h4>Επόμενο</h4>
     <p>Αποκτήστε τον κώδικα ενσωμάτωσης για να δημοσιεύσετε το γράφημα κάνοντας κλικ στο κουμπί <img class="icon-insert" src="./embed_button.svg" /> στο επάνω δεξιό μέρος του πίνακα γραφημάτων.</p>
     <h4>Συνεχίστε τα μαθήματα:</h4>
     <ol>
-      <li><a href="/lessons/el/prosarmosmenes-eidopoihseis/">Παρακολουθήστε δεδομένα και ορίστε προσαρμοσμένες ειδοποιήσεις</a></li>
-      <li><a href="/lessons/el/kathariste-vromika-dedomena/">Συλλέξτε περιεχόμενο από μια ιστοσελίδα και παρακολουθήστε το</a></li>
+      <li><a href="prosarmosmenes-eidopoihseis">Παρακολουθήστε δεδομένα και ορίστε προσαρμοσμένες ειδοποιήσεις</a></li>
+      <li><a href="kathariste-vromika-dedomena">Συλλέξτε περιεχόμενο από μια ιστοσελίδα και παρακολουθήστε το</a></li>
```
Comment (giorgio93p) [inline server/lessons/el/analysh-dedomenwn-twitter.html]: The text in `<a href="kathariste-vromika-dedomena"></a>` should be "Καθαρίστε και τυποποιήστε δεδομένα"

File: server/lessons/en/analyze-live-twitter.html [hunk 4]
```diff
@@ -145,10 +145,10 @@ <h4>Next</h4>
     <h4>Continue learning:</h4>
     <ol>
       <li>
-        <a href="/lessons/custom-alerts/">Monitor data and set custom alerts</a>
+        <a href="custom-alerts">Monitor data and set custom alerts</a>
       </li>
       <li>
-        <a href="/lessons/clean-and-standardize/">Scrape a webpage to extract or monitor content</a>
+        <a href="clean-and-standardize">Scrape a webpage to extract or monitor content</a>
```
Comment (giorgio93p) [inline server/lessons/en/analyze-live-twitter.html]: The text should be "Clean and standardize data"

File: server/lessons/en/scrape-multiple-pages.html [hunk 5]
```diff
@@ -6,7 +6,7 @@ <h1>VII. Scrape multiple pages on one website</h1>
 <section class="fullscreen">
   <h2>Overview</h2>
   <p>
-    Suppose we want to scrape this list of audit reports from the <a href="https://leg.colorado.gov/audit-search" class="highlight" target="_blank">Colorado state government</a>. We've learned in the tutorial <a href="https://app.workbenchdata.com/lessons/scrape-using-xpath" class="highlight">Scrape a webpage to extract or monitor content</a> to isolate the titles, release date and report numbers using Selector Gadget.
+    Suppose we want to scrape this list of audit reports from the <a href="https://leg.colorado.gov/audit-search" class="highlight" target="_blank">Colorado state government</a>. We've learned in the tutorial <a href="scrape-using-xpath" class="highlight">Scrape a webpage to extract or monitor content</a> to isolate the titles, release date and report numbers using Selector Gadget.
```
Comment (giorgio93p) [inline server/lessons/en/scrape-multiple-pages.html]: This change hasn't been ported to Greek

File: server/lessons/en/scrape-multiple-pages.html [hunk 6]
```diff
@@ -77,22 +77,16 @@ <h2>Extract data from multiple HTML pages</h2>
   <p>
     We still need to extract the information we want from all this source code. We can do so with <b>HTML to Table</b>. It will automatically combine the results from all pages into one table.
   <p class="instruction-detail">
-    Learn how to use <b>HTML to Table</b> and XPath in this tutorial:<br> <a href="https://app.workbenchdata.com/lessons/scrape-using-xpath" class="highlight">Scrape a webpage to extract or monitor content</a>
+    Learn how to use <b>HTML to Table</b> and XPath in this tutorial:<br><a href="scrape-using-xpath" class="highlight">Scrape a webpage to extract or monitor content</a>
```
Comment (giorgio93p) [inline server/lessons/en/scrape-multiple-pages.html]: This change hasn't been ported to Greek (the Greek file does not have `https://app.workbenchdata.com` but has `/lessons/el/`)

File: server/lessons/en/scrape-multiple-pages.html [hunk 7]
```diff
@@ -77,22 +77,16 @@ <h2>Extract data from multiple HTML pages</h2>
...
-        </li>
-        <li>
-          <a class="highlight" href="/lessons/custom-alerts/">Set up custom alerts</a> to receive emails if conditions are met.
-        </li>
-        <li>
-          <a class="highlight" href="/lessons/clean-and-standardize/">Clean and standardize data</a>.
-        </li>
-      </ul>
+    <ul>
+      <li><a href="custom-alerts">Scrape a webpage to extract or monitor content</a>.</li>
```
Comment (giorgio93p) [inline server/lessons/en/scrape-multiple-pages.html]: The first link should point to `scrape-using-xpath`

**Comment by adamhooper** [comment]
Thank you! I'll make all these changes, since it's late in the day for you.

File: server/lessons/en/analyze-live-twitter.html (hunk 4 above)
Comment (adamhooper) [reply server/lessons/en/analyze-live-twitter.html]: Actually, I think it makes more sense to link to `scrape-using-xpath`

**Comment by adamhooper** [comment]
@giorgio93p Could you please look at my changes once more to make sure I didn't do something stupid?

**Pull request merged**